CONTRIBUTING.md

```diff
@@ -156,20 +156,12 @@ docker push ${REGISTRY}/localbuildcrd:yourtag
 ```
 
 ### Deploy using Dev Helmfile
-#### Update dev.helmfile.yaml
-Replace `repository`, `crdRepository`, and `tag` with previously built images:
-```yaml
-- name: image.repository 
-  value: <YOUR RATIFY IMAGE REPOSITORY NAME>
-- name: image.crdRepository
-  value: <YOUR RATIFY CRD IMAGE REPOSITORY NAME>
-- name: image.tag
-  value: <YOUR IMAGES TAG NAME>
-```
+
+Development charts + images are published weekly and latest versions are tagged with rolling tags referenced in dev helmfile.
 
 Deploy to cluster:
 ```bash
-helmfile sync -f dev.helmfile.yaml
+helmfile sync -f git::https://github.com/deislabs/ratify.git@dev.helmfile.yaml
```

Comment:
since we deleted line 159-168, wonder how developers deploy their local changes via helmfile?

Reply:
good point. I'll add that back in the section above which describes the flow of building your own dev images